fabtests/scripts/runmultinode.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 Options=$(getopt --options h:,n:,p:,I:,-x:,z: \
-		  		--longoptions hosts:,processes-per-node:,provider:,xfer-method:,iterations:,ci:,cleanup,help \
+				--longoptions hosts:,processes-per-node:,provider:,xfer_method:,iterations:,ci:,cleanup,help \
```

Comment:
Why would `--xfer-method` cause problem?  Did you use `-` instead of `--` (notice that `-x` is a valid option).

Reply:
The script gets interpreted as  "..... -p 'verbs;ofi_rxm' -x -method -z full_mesh -I 1 -T....". I guess i misunderstood the error - The '-' is actually causing a misinterpretation of the of the bash variable xfer-method. this can be fixed using curly braces. However, "-" is not recommended in variable names to avoid such errors so i think using "_" is still better. I will correct the reason in the commit message